account_multicurrency_revaluation/demo/account_demo.xml

```diff
@@ -0,0 +1,86 @@
+        <field name="user_type_id" ref="account.data_account_type_current_assets" />
+    </record>
+    <record id="acc_prov_bs_loss" model="account.account">
+        <field name="name">Prov B.S Loss</field>
+        <field name="code">XPBS001</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets" />
+    </record>
+    <record id="acc_prov_bs_gain" model="account.account">
+        <field name="name">Prov B.S Gain</field>
+        <field name="code">XPBS002</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets" />
+    </record>
+    <record id="acc_prov_pl_loss" model="account.account">
+        <field name="name">Prov P L Loss</field>
+        <field name="code">XPPL001</field>
+        <field name="user_type_id" ref="account.data_account_type_expenses" />
+    </record>
+    <record id="acc_prov_pl_gain" model="account.account">
+        <field name="name">Prov P L Gain</field>
+        <field name="code">XPPL002</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue" />
+    </record>
+    <record id="demo_acc_receivable" model="account.account">
+        <field name="name">Account Receivable</field>
+        <field name="code">accrec</field>
+        <field name="user_type_id" ref="account.data_account_type_receivable" />
+        <field name="currency_revaluation">True</field>
+        <field name="reconcile">True</field>
+    </record>
+    <record id="demo_acc_revenue" model="account.account">
+        <field name="name">Account Revenue</field>
+        <field name="code">accrev</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue" />
+        <field name="currency_revaluation">True</field>
+    </record>
+    <record id="demo_acc_liquidity_usd" model="account.account">
+        <field name="name">Account Liquidity USD</field>
+        <field name="code">accliq_usd</field>
+        <field name="user_type_id" ref="account.data_account_type_liquidity" />
+        <field name="currency_revaluation">True</field>
+        <field name="currency_id" ref="base.USD" />
+    </record>
+    <record id="demo_acc_liquidity_gbp" model="account.account">
+        <field name="name">Account Liquidity GBP</field>
+        <field name="code">accliq_gbp</field>
+        <field name="user_type_id" ref="account.data_account_type_liquidity" />
+        <field name="currency_revaluation">True</field>
+        <field name="currency_id" ref="base.GBP" />
+    </record>
+    <record id="reval_journal" model="account.journal">
+        <field name="name">Revaluation Journal</field>
+        <field name="code">RJ</field>
+        <field name="type">general</field>
+    </record>
+    <record id="bank_journal_usd" model="account.journal">
+        <field name="name">Bank Journal USD</field>
+        <field name="code">BJ USD</field>
+        <field name="type">bank</field>
+        <field name="currency_id" ref"base.USD"/>
```

Comment:
Please rebase on grindtildeath:10.0-mig-account_multicurrency_revaluation to correct ref=